packages/web-components/fast-foundation/src/design-system-provider/design-system-provider.ts

```diff
@@ -416,7 +416,7 @@ export class DesignSystemProvider extends FASTElement
 /**
  * Defines a design-system-provider custom element
  */
-export function designSystemProvider(name: string) {
+export function designSystemProvider(name: string, styles: ElementStyles | undefined) {
```

Comment:
@nicholasrice and @EisenbergEffect want to bring attention to this API change as part of this. Because we aren't passing styles I needed to add that as part of the decorator here for composing the custom element.

Reply:
Do we need to pass the template here also?

Rob and I discussed potentially opening this up to have the same API as `@customElement` - this might be the right time to make that change.

Reply:
In that case, I think we should do that. I'll update.

Reply:
Updated!